mp/src/game/shared/momentum/mom_system_gamemode.cpp

```diff
@@ -120,6 +126,11 @@ void CGameMode_RJ::SetGameModeVars()
     sv_ground_trigger_fix.SetValue(false); // MOM_TODO Remove when bounce triggers have been implemented
 }
 
+float CGameMode_RJ::GetJumpFactor()
+{
+    return sqrtf(52.200625f * 2.0f * 800.0f);
```

Comment:
Using the actual jump velocity for jump and ahop would make more sense since they are already defined using a velocity and not a jump height like CS:S.

Reply:
Ah, changed those back then :)